configs/config.json.example

```diff
@@ -13,6 +13,9 @@
     "test": false,
     "location_cache": true,
     "distance_unit": "km",
+    "pokestop_cooldown": "300",
```

Comment:
It does make sense.  The area where I run my bot has Pokestops evenly spaced and close together.  So what ends up happening is my just goes around in a circle, I end up with a ton of items and very few pokemon.  By setting the timeout to be longer, it forces the bot to find a fort that is not in timeout.

__init__.py: line 190: forts = filter(lambda x: x["id"] not in self.fort_timeouts, forts)

This allows the user to create a larger search area as the bot will now have to find forts farther away because it thinks all the ones close are in timeout.